internal/app/tfsec/block/hclblock.go

```diff
@@ -197,6 +197,9 @@ func (block *HCLBlock) GetAttributes() []Attribute {
 }
 
 func (block *HCLBlock) GetAttribute(name string) Attribute {
+	if strings.Contains(name, "/") {
```

Comment:
Can block names definitely not contain `/`? If they can we could just ask that people call `GetNestedAttribute` directly.

Reply:
I don't think so, but I agree it should be an explicit action so have fixed